src/audio/google_hotword_detect.c

```diff
@@ -200,16 +200,20 @@ static int ghd_setup_model(struct comp_dev *dev)
 	struct comp_data *cd = comp_get_drvdata(dev);
 	void *model;
 	size_t size = 0;
-	uint32_t crc = 0;
 	int ret;
 
-	model = comp_get_data_blob(cd->model_handler, &size, &crc);
+	/* Avoid the CRC calculation since it takes too long and causes XRUN.
```

Comment:
Fwiw, IPC work (and hence the CRC) will be preemptable with Zephyr rtos - so other work will have priority and run as normal.

Reply:
Good to know, once we have a platform we can test zerphyr with ill add a compile switch here. @bzhg FYI